File: 1_File_Creation/file_create.py
# Ask the user to enter the file name
str1 = input("File name: ")

# Open the file in write mode ("w" means it will create or overwrite the file)
with open(str1, "w") as f:
    # Ask the user to enter the content they want to write in the file
    str2 = input("Content you want to write: ")

    # Write the user's input into the file
    f.write(str2)

    # Show confirmation message
    print("File created and writed successfully")

File: 1_File_Creation/file_create.cpp
#include <iostream>   // For input and output
#include <fstream>    // For file operations

using namespace std;

int main()
{
    string str1, str2;  // str1: file name, str2: content to write in file

// Ask user to enter a file name (without spaces)
    cout << "Enter file name (without_Spaces):";
    cin >> str1;

    cin.ignore(); // Clear the input buffer before taking getline input

// Create and open the file using the given file name
    ofstream out(str1);

// Ask user for the content (syntax) to write in the file
    cout << "syntax:";
// Read a full line
    getline(cin, str2);

    // Write the content to the file
    out << str2;

    // Confirmation message
    cout << "File created and writed successfully\n";

    // File will be automatically closed when 'out' goes out of scope
}
